### DISCUSSION TITLE
Adding FAiSS?

### DISCUSSION THREAD

**Discussion by kristianjackson** [FlowiseAI/Flowise#108] [Ideas]
I noticed that FAiSS is not an option for vector stores. Is there a timeline for introducing it?

**Comment by HenryHengZJ** [comment]
As Flowise is built on top of LangChainJS, FAiSS is not yet available.

Here's a list of available integrations to date: https://js.langchain.com/docs/modules/indexes/vector_stores/integrations/

**Reply by HenryHengZJ** [reply]
PR coming in from https://github.com/hwchase17/langchainjs/pull/685
